packages/react-components/src/components/ChatMessage/ChatMessageComponentWrapper.tsx

```diff
@@ -65,8 +67,8 @@ export type ChatMessageComponentWrapperProps = _ChatMessageProps & {
    * Optional callback to define custom actions for attachments.
    */
   actionsForAttachment?: (attachment: AttachmentMetadata, message?: ChatMessage) => AttachmentMenuAction[];
-  /* @conditional-compile-remove(rich-text-editor) */
-  richTextEditor?: boolean;
+  /* @conditional-compile-remove(rich-text-editor) @conditional-compile-remove(rich-text-editor-image-upload) */
```

Comment:
Should the second CC flag be removed, as the `onPaste` should be handled in the type definition?